src/Services/People.ts

```diff
@@ -0,0 +1,53 @@
+import axios from "axios";
+import Cookies from "universal-cookie";
+import config from "../config";
+import { ICreatePerson, IUpdate } from "../Interfaces/Person";
+
+const peopleApiRoute = `${config.API_BASE_URL}/person`;
+
+axios.interceptors.request.use((config) => {
+  if (!config.headers) config.headers = {};
+
+  const cookies = new Cookies();
+  config.headers!["Authorization"] = cookies.get("jwt");
```

Comment:
```suggestion
  config.headers!["Authorization"] = `Bearer ${cookies.get("jwt")}`;
```

Reply:
you said not to use a bearer and now you do :(

Reply:
I said use Bearer in the Authorization header, not in the cookie